### DISCUSSION TITLE
Appwrite ram to website ratio

### DISCUSSION THREAD

**Discussion by cambo2015** [appwrite/appwrite#1191] [General]
Hey! 🎉 I'm wondering how many websites could I run in 1 Digital Ocean server with Appwrite's minimum ram requirements? Is it a  2gb ram :1 Website ratio?

**Comment by cambo2015** [comment]
> Hey! 🎉 I'm wondering how many websites could I run in 1 Digital Ocean server with Appwrite's minimum ram requirements? Is it a  2gb ram :1 Website ratio?

Not sure if this was clear enough.
Basically, if I just had one cpu core with 2gb ram how many websites could I run on it via appwrite? Thanks in advance

**Reply by eldadfux** [reply]
Hey @cambo2015 it really depends on your API usage much more than the number of websites or projects you run on the instance.

**Reply by cambo2015** [reply]
Thank you! That makes sense.

**Comment by Duramax2018** [comment]
What do you mean

**Reply by cambo2015** [reply]
What he means, as I understand it, is that the more traffic your website gets, the more ram you need. So the way I had thought about it was off

**Comment by Duramax2018** [comment]
Thanks, I'll check it out.
